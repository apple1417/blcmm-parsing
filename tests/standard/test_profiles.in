<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="a"/>
			<profile name="-" current="true"/>
			<profile name="a b"/>
			<profile name="-_-_-_-_-"/>
			<profile name="BLCMM"/>
			<profile name="exec a"/>
			<profile name="say 11111111111111111111"/>
		</profiles>
	</head>
	<body>
		<category name="root">
		</category>
	</body>
</BLCMM>

#Commands:

#Profile notice:
say Executed your mod with the following profile: -.

