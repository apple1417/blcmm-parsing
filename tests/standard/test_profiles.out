<BLCMM v="1">
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="a"/>
			<profile name="-" current="true"/>
			<profile name="a b"/>
			<profile name="-_-_-_-_-"/>
			<profile name="BLCMM"/>
			<profile name="exec a"/>
			<profile name="say 11111111111111111111"/>
		</profiles>
	</head>
	<body>
		<category name="root">
		</category>
	</body>
</BLCMM>
